install_config/aggregate_logging.adoc

```diff
@@ -440,8 +440,19 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-To scale Elasticsearch up, create more deployments. The deployer provides
-a template to create Elasticsearch deployments:
+If you need to scale up the number of Elasticsearch instances your cluster uses
+it is not as simple as changing the number of replicas. This is due to the nature
+of Persistent volumes and how Elasticsearch is configured to store its data.
+
+Instead, you will need to create a Deployment Configuration for each replica.
+
+The Deployer created templates during installation with the Elasticsearch
+configurations provided to it: `logging-es-template` and `logging-es-ops-template`
+if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+
+To create an additional Deployment Configuration, run the following command against
+the Elasticsearch cluster you'd like to scale up for (`logging-es-template` or
+`logging-es-ops-template`).
```

Comment:
(sorry for my lateness) yes, that sounds right. 4 and 5 should be close together in time :)